base_joint: 3.14
shoulder_joint: 3.6778498611256336
shoulder_shadow: 3.676315505739265
elbow_joint: 1.155369605935587
wrist_joint: 3.2296268563296833
gripper_joint: 3.1214